<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManagerForm</class>
 <widget class="QWidget" name="OrderManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,2,1,1,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>주문정보관리</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearbutton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="image.qrc">
             <normaloff>:/images/erase.png</normaloff>:/images/erase.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>주문정보관리</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="ClientProductSearch">
         <item row="1" column="0">
          <widget class="QLineEdit" name="ClientLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="cbutton">
           <property name="text">
            <string>ClientSearch</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="ProductLineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pbutton">
           <property name="text">
            <string>ProductSearch</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeWidget" name="clienttreeWidget">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Phone Number</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Address</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="producttreeWidget">
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Price</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Stock</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="OrderformLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="idLabel">
           <property name="text">
            <string>&amp;OrderNum</string>
           </property>
           <property name="buddy">
            <cstring>idLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="idLineEdit">
           <property name="text">
            <string>Auto-generated</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>&amp;Client</string>
           </property>
           <property name="buddy">
            <cstring>ClientLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="CIDlineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="phoneNumberLabel">
           <property name="text">
            <string>&amp;Product</string>
           </property>
           <property name="buddy">
            <cstring>ProductLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="PIDlineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>&amp;Stock</string>
           </property>
           <property name="buddy">
            <cstring>StockLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="StockLineEdit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="placeholderText">
            <string>Only Number</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="pricelineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addPushButton">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifyPushButton">
             <property name="text">
              <string>&amp;Modify</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="addresslineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Click_List</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>&amp;Search</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>OrderNum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Client (Name or ID)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Product (Name or ID)</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="searchPushButton">
              <property name="text">
               <string>S&amp;earch</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="treewidgetLayout" stretch="7">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>75</number>
         </attribute>
         <column>
          <property name="text">
           <string>OrderNum</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ClientName</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Product</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Stock</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sum</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
